Package.swift

```diff
@@ -359,15 +384,17 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         // The 'swift-argument-parser' version declared here must match that
         // used by 'swift-driver' and 'sourcekit-lsp'. Please coordinate
         // dependency version changes here with those projects.
-        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "0.4.3")),
+        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: Version(0, 4, 3))),
```

Comment:
What's the reason for this formatting change here?

Reply:
I thought by using `Version`'s initialiser directly, instead of a string literal, we could reduces a tiny (probably negligible) amount of dependency resolution work. But this probably doesn't bring much benefit. I can change it back if string literals are preferred.

Reply:
I changed all versions back to string literals.